Orange/widgets/gui.py

```diff
@@ -552,6 +552,14 @@ def widgetBox(widget, box=None, orientation='vertical', margin=None, spacing=4,
     return b
 
 
+def horizontalBox(*args, **kwargs):
+    return widgetBox(orientation="horizontal", *args, **kwargs)
+
+
+def verticalBox(*args, **kwargs):
```

Comment:
Congratulations!

Please make these [`hBox` and `vBox`](https://www.google.com/search?q=hbox+vbox).